internal/store/cluster_installation.go

```diff
@@ -166,6 +164,23 @@ func (sqlStore *SQLStore) RecoverClusterInstallation(clusterInstallation *model.
 	return nil
 }
 
+// DeleteInActiveClusterInstallationByClusterID marks the inactive cluster installation as deleted for a given cluster, but does not remove
+// the record from the database.
+func (sqlStore *SQLStore) DeleteInActiveClusterInstallationByClusterID(clusterID string) error {
+	_, err := sqlStore.execBuilder(sqlStore.db, sq.
+		Update("ClusterInstallation").
+		Set("State", model.ClusterInstallationStateDeletionRequested).
+		Where("ClusterID = ?", clusterID).
+		Where("IsActive = ?", false).
+		Where("DeleteAt = 0"),
+	)
+	if err != nil {
+		return errors.Wrap(err, "failed to mark inactive cluster installation as deleted")
```

Comment:
@Szymongib Do we want to apply "initial capital" for these error messages as well ? Currently all the existing code is following the "init lower case" pattern. I'm afraid that making the new methods with "init upper case"  will create inconsistency unless we r determined to migrate all the existing error messages to "init upper case" gradually .

Reply:
No, the errors should stay lower case, only the log messages should start with upper.